caas/kubernetes/provider/bootstrap.go

```diff
@@ -97,6 +123,11 @@ func newcontrollerStack(stackName string, storageClass string, broker *kubernete
 	agentConfig.SetStateServingInfo(si)
 	pcfg.Bootstrap.StateServingInfo = si
 
+	// we use controller name to name controller namespace in bootstrap time.
+	if err = setControllerNamespace(pcfg.ControllerName, broker); err != nil {
+		return nil, errors.Trace(err)
+	}
+
 	cs := controllerStack{
 		stackName:   stackName,
 		namespace:   broker.GetCurrentNamespace(),
```

Comment:
It seems a bit wrong to have namespace here as well as on the broker below.

Reply:
yes, i will remove `controllerStack.namespace`.